### ISSUE TITLE
Training day-night

### ISSUE THREAD

**Issue by rudolphyo** [yenchenlin/pix2pix-tensorflow#19]
Had anyone tried to reproduce results of day to night?

The problem is that i get very poor results. I take dataset mentioned in paper and make folders A and B in this way : take strong day photos(daylight > 0.8) for A and strong night photos (night > 0.6) for B for each webcam. Then i made some data augmentation through rotating/mirroring of images. After that I combined them with script and change batch_size from 1 to 4 and epochs from 200 to 17 as in paper. 

Thank you!

**Comment by ppwwyyxx** [comment]
Did you make sure to apply identical augmentations to each training/testing pair?

**Comment by rudolphyo** [comment]
@ppwwyyxx yeah, sure. If in A i have 90 left rotated image, so in B it is the same, but night one.
I can share my script that I`ve used for augmentation

**Comment by AshuKV** [comment]
@rudolphyo Can you please share your github link or your training and testing scripts for your datasets ?
And how did you train it?

**Comment by fabiosoft** [comment]
I get a very blurry and weird artifacts in results. 
`--batch_size 4 --preprocess scale_width_and_crop --load_size 754 --crop_size 512`
